src/foundry/common/prosemirror/paste-transformer.d.mts

```diff
@@ -2,14 +2,15 @@ import type { Schema, Slice } from "prosemirror-model";
 import type ProseMirrorPlugin from "./plugin.d.mts";
 import type { Plugin } from "prosemirror-state";
 import type { EditorView } from "prosemirror-view";
+import type { EmptyObject } from "../../../types/utils.d.mts";
 
 export default ProseMirrorPasteTransformer;
 
 /**
  * A class responsible for applying transformations to content pasted inside the editor.
  */
 declare class ProseMirrorPasteTransformer extends ProseMirrorPlugin {
-  static override build(schema: Schema, options?: Record<string, unknown>): Plugin;
+  static override build(schema: Schema, options?: EmptyObject | undefined): Plugin;
```

Comment:
So `| undefined` in an optional function parameter has no effect. The only difference is with object properties. So you don't need `| undefined` here or in other functions.